### PR TITLE
missed punctuation

### PR DESCRIPTION
**Pull request by aj07**
there are some places where "." and ":" missed. Please, add this.

### CODE REVIEW

File: docs/debugging-openshift.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ This document contains some tips and suggestions for troubleshooting an OpenShif
 System Environment
 ------------------
 
-1. Run as root
+1. Run as root.
```
Comment (pweil-) [inline docs/debugging-openshift.md]: Thanks for the update.  It looks like this is missing from the next 2 numbered items as well.  Please update for those.

**Comment by pweil-** [comment]
[merge]

**Comment by pweil-** [comment]
[merge]

**Comment by aj07** [comment]
@pweil- Why test is failing? How can I put it on re-check?

**Comment by pweil-** [comment]
> Why test is failing? How can I put it on re-check?

@aj07 known flake (https://github.com/openshift/origin/issues/12236).  I'll make sure it has the correct tags for testing/merging.

**Pull request merged**